packages/trader/src/Modules/MT5/Helpers/constants.js

```diff
@@ -17,9 +17,10 @@ const getPlatformMt5DownloadLink = () => {
     }
```

Comment:
@reza-fs these also need to return differently based on EU, please also get the latest changes from dev as non-eu links are outdated

Reply:
Will do.